cicd/pipelines/using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id="using-pipelines-as-code"]
+= Using Pipelines as Code
+include::_attributes/common-attributes.adoc[]
+:context: using-pipelines-as-code
+
+toc::[]
+
+:FeatureName: Pipelines as Code
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+With {pac}, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by a source code push or a pull request for the configured Git repository, the feature runs the pipeline and reports the status.   
+
+[id="pac-key-features"]
+== Key features
+{pac} supports the following features:
+
+* Pull request status. When iterating over a pull request, the status and control run on the platform hosting the Git repository.
+* GitHub Checks API to set the status of a pipeline run, including rechecks.
```

Comment:
IBM SG, parallelism. Please restate this so its structure parallels that of the other feature bullets and helps users avoid confusion about its meaning.

Reply:
```suggestion
* GitHub Checks API, which you can use to set the status of a pipeline run, including rechecks.
```

Reply:
IBM SG: 
- You can introduce procedures with a sentence, an infinitive phrase, or a heading. Make list items grammatically parallel. For example, do not use a mix of phrases and sentences or a mix of passive and
active voice in the same list.
- Make sure that the elements of a sentence are parallel. Words, phrases, and clauses should be grammatically equal.